test/test_helper.rb

```diff
@@ -69,4 +68,13 @@ class ApplicationIntegrationTest < ActionDispatch::IntegrationTest
 
 class EppTestCase < ActionDispatch::IntegrationTest
   include Assertions::EppAssertions
+
+  def return_xml_domain_schema_version(response_xml)
```

Comment:
I'd suggest to modify this method with adding filename prefix, ATM it's just hardcoded as a `domain`, so we could not use this method to check for, say, contact schema version.